server.properties

```diff
@@ -0,0 +1,39 @@
+#Minecraft server properties
+#(File modification datestamp)
+spawn-protection=16
+max-tick-time=60000
+generator-settings=
+force-gamemode=false
+allow-nether=true
+gamemode=0
+enable-query=false
+player-idle-timeout=0
+difficulty=1
+spawn-monsters=true
+op-permission-level=4
+resource-pack-hash=
+announce-player-achievements=true
+pvp=true
+snooper-enabled=true
+level-type=DEFAULT
+hardcore=false
+enable-command-block=true
+max-players=20
+network-compression-threshold=256
+max-world-size=29999984
+server-port=25565
+server-ip=
+spawn-npcs=true
+allow-flight=false
+level-name=world
+view-distance=10
+resource-pack=
+spawn-animals=true
+white-list=false
+generate-structures=true
+online-mode=false
+max-build-height=256
+level-seed=1122334455667788
+use-native-transport=true
+motd=A Minecraft Server made in Rust!
+enable-rcon=false
```

Comment:
The nice thing about having both `server.properties` and `favicon.png` on root is that we can do `cargo run` and it just works.

We could drop the `server.properties` file but keep favicon maybe with the project's logo (when we have one).